src/Sulu/Component/Webspace/Tests/Unit/Manager/WebspaceManagerTest.php

```diff
@@ -501,8 +508,10 @@ public function provideFindPortalInformationByUrl()
 
     /**
      * @dataProvider provideFindPortalInformationByUrl
+     *
+     * @param bool $shouldFind
      */
-    public function testFindPortalInformationByUrlWithInvalidSuffix($url, $shouldFind): void
+    public function testFindPortalInformationByUrlWithInvalidSuffix(string $url, $shouldFind): void
```

Comment:
This is not a bc break because the function where those variables are used already have a return type of array. So the only thing is we fail the code faster on broken types.

Reply:
yeah normally parameters types are bc breaks but in tests we don't need to take care about bc and can set them.

Reply:
So in tests "BC breaking behavior" (like adding parameters and return types) is okay in test methods?

Reply:
@mamazu yes totally in tests we don't need to take care of the bc breaks, only the `Testing` namespace of the SuluTestBundle is the only one which part of the public namespace.

We already added and fixed some types in tests via rector here:

 - https://github.com/sulu/sulu/pull/6845
 - https://github.com/sulu/sulu/pull/6839
 
If we do that kind of massive manipulation I would prefer todo it on 2.4 branch or for PHP 8.0 changes on 2.5 avoids that we have problems in backmerge a bugfix with a test.

Reply:
Okay, I will keep that in mind for further type changes.